xAPI-Communication.md

```diff
@@ -224,21 +224,15 @@ __Query string parameters__:
 __Attachments__: Note that due to issues relating to encoding, it is not possible to send 
 binary data attachments using this syntax. See [4.1.11. Attachments](#attachments) 
 
-* The LRS MUST support the syntax above.
+* The LRS MUST support the syntax above.  See [Appendix C: Cross Domain Request Example](#Appendix3C) for an example. 
```

Comment:
The added text should be on a new line otherwise it sounds like the example is just for LRSs.